.github/workflows/trigger_ci.yaml

```diff
@@ -0,0 +1,252 @@
+name: Trigger next gating
+
+on:
+  workflow_dispatch:
+    inputs:
+      base_sha:
```

Comment:
why `base_sha` and `head_sha` are defined as inputs?

Reply:
it's just a name of the field , when you trigger manually you need to provide the SHA for both

Reply:
`git diff --name-only ${{ github.event.pull_request.base.sha }} ${{ github.event.pull_request.head.sha }}` - not used here